README.md

```diff
@@ -1,6 +1,7 @@
 # Giraf Backend
 
 ![Build Status](https://github.com/aau-giraf/web-api/workflows/.NET%20Core/badge.svg)
```

Comment:
Should this be changed since the workflow name was changed?

Reply:
Good point. Fixed in 0152dd9.

Reply:
Woops, missed the URL - fixed in aa467a4 as well